graylog2-web-interface/src/stores/PaginationTypes.ts

```diff
@@ -19,6 +19,7 @@ import { $PropertyType } from 'utility-types';
 
 import type { AdditionalQueries } from 'util/PaginationURL';
 
+/* eslint-disable camelcase */
```

Comment:
@kmerz Was about to merge the PR, but just saw this comment is no longer needed. It is a nitpick, but could you remove the comment, now where we disabled the rule and removed all similiar comments?